src/main/java/racingcar/RandomMove.java

```diff
@@ -0,0 +1,31 @@
+package racingcar;
+
+import java.util.Random;
+
+public class RandomMove {
+    private static final int STANDARD_MOVEMENT = 4;
+    private static final int RANDOM_BOUND = 10;
+
+    private final Random random = new Random();
+
+    private final int standard;
+    private final int bound;
+
+    public RandomMove() {
+        this(STANDARD_MOVEMENT, RANDOM_BOUND);
+    }
+
+    public RandomMove(int standard, int bound) {
+        this.standard = standard;
+        this.bound = bound;
+    }
+
+    public boolean canMove() {
+        int randomNum = random.nextInt(bound);
+        if(randomNum >= standard) {
+            return true;
+        }
+        return false;
```

Comment:
아래와 같이 수정해 줄 수 있겠네요!

```suggestion
return randomNum >= standard;
```